src/frontend/qt_sdl/main.cpp

```diff
@@ -2188,101 +2287,128 @@ bool MainWindow::preloadROMs(QStringList file, QStringList gbafile, bool boot)
     return true;
 }
 
-QString MainWindow::pickFileFromArchive(QString archiveFileName)
+QStringList MainWindow::splitArchivePath(const QString& filename, bool useMemberSyntax)
 {
-    QVector<QString> archiveROMList = Archive::ListArchive(archiveFileName);
+    if (filename.isEmpty()) return {};
 
-    QString romFileName = ""; // file name inside archive
-
-    if (archiveROMList.size() > 2)
+#ifdef ARCHIVE_SUPPORT_ENABLED
+    if (useMemberSyntax)
     {
-        archiveROMList.removeFirst();
+        printf("Warning: use the a.zip|b.nds format at your own risk!\n");
```

Comment:
doesn't melonDS internally rely on the |, e.g. for the recent rom list, so it would print that for no good reason?

Reply:
This message was in a part of `src/frontend/qt_sdl/CLI.cpp` which I have replaced and I didn’t initially find a good place to put it in the new code. Fixed in b2d2cf3d1abb40594303285d2d48454500a3b908.

Reply:
shouldn't this line be removed now when you added
```c++
if (memberSyntaxUsed) printf("Warning: use the a.zip|b.nds format at your own risk!\n");
```
?

Reply:
Seems I just forgot to remove it. Fixed in ca75766c9c821b17661a05be2de88de0c3c16821.